### PR TITLE
Colab for sparse transferring dense Hugging Face models

### PR DESCRIPTION
**Pull request by InquestGeronimo**
colab: https://colab.research.google.com/drive/1I5ez6ZpdT0K-yo7l9AXrrJ7tIFoEP8Jv#scrollTo=Xu84qy9Tileq

### CODE REVIEW

File: integrations/huggingface-transformers/tutorials/Sparse_Transfer_HF_Dense_Model.ipynb [hunk 1]
```diff
@@ -0,0 +1,5775 @@
...
+        "!pip install sparseml[torch]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LGlWL__nGqb7"
+      },
+      "source": [
+        "Run the following the CLI command to initiate the sparse transfer learning. The dense model, as seen in the `distill_teacher` argument, will transfer its knowledge onto a 6 layer pruned quantized bert base student model, as seen in the `model_name_or_path` argument. When doing sparse transferring, your dense model's architecture needs to match the architecture of the student model, which in this case is bert base uncased.\n",
```
Comment (anmarques) [inline integrations/huggingface-transformers/tutorials/Sparse_Transfer_HF_Dense_Model.ipynb:278]: The dense model doesn't need to match the architecture of the student model, at least for this example. For the standard form of distillation (the one used here) the only thing that needs to match between student and teacher models is the structure of the output

File: integrations/huggingface-transformers/tutorials/Sparse_Transfer_HF_Dense_Model.ipynb [hunk 2]
```diff
@@ -0,0 +1,5775 @@
...
+            "    )\n",
+            "    (encoder): BertEncoder(\n",
+            "      (layer): ModuleList(\n",
+            "        (0): BertLayer(\n",
+            "          (attention): BertAttention(\n",
+            "            (self): BertSelfAttention(\n",
+            "              (query): QuantWrapper(\n",
+            "                (quant): QuantStub(\n",
+            "                  (activation_post_process): FakeQuantize(\n",
+
```
Comment (anmarques) [inline integrations/huggingface-transformers/tutorials/Sparse_Transfer_HF_Dense_Model.ipynb:5422]:
```suggestion
        "Let's do the same to the dense model. First we'll download it to a directory we'll call `dense_model`, and then we'll export it to ONNX:"
```

**Review by anmarques — CHANGES_REQUESTED** [review]
Very cool. Small correction is needed about the teacher and student model architectures needing to match

File: integrations/huggingface-transformers/tutorials/Sparse_Transfer_HF_Dense_Model.ipynb (hunk 1 above)
Comment (InquestGeronimo) [reply integrations/huggingface-transformers/tutorials/Sparse_Transfer_HF_Dense_Model.ipynb:278]: thank you

**Review by anmarques — APPROVED** [review]

**Review by mgoin — APPROVED** [review]

**Pull request merged**